src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const listToSort = document.querySelector('ul');
+
+function strToNum(string) {
+  return Number(string.replace('$', '').replace(',', ''));
+}
+
+const sortList = (list) => {
+  const copyList = [...list.children];
+
+  copyList.sort((a, b) =>
+    strToNum(b.dataset.salary) - strToNum(a.dataset.salary));
+  copyList.forEach(item => list.append(item));
+};
+
+const getEmployees = (list) => {
+  const copyList = [...list.children];
+
+  copyList.map(item => ({
```

Comment:
The map() method will create a copy anyway. So, you don't need to create a new copy